app/models/mhv_account.rb

```diff
@@ -120,8 +127,12 @@ def create_mhv_account!
 
   def upgrade_mhv_account!
     if may_upgrade?
+      StatsD.increment("#{STATSD_ACCOUNT_UPGRADE_KEY}.total")
+
       client_response = mhv_ac_client.post_upgrade(params_for_upgrade)
       if client_response[:status] == 'success'
+        StatsD.increment("#{STATSD_ACCOUNT_UPGRADE_KEY}.success")
+
```

Comment:
I think my point is that it should be consistent with how we're doing it for account creation. Account creation is different from Account Upgrading in that with account creation we skip it altogether if a mhv correlation id already exists, whereas for upgrading we attempt to do the upgrade only because we have no other way of determining whether or not the account is upgraded or not -- but perhaps it shouldn't count towards the total unless the account actually required being upgraded (in an effort to keep things consistent with how we do account creation).